gcloud/storage/acl.py

```diff
@@ -345,77 +348,64 @@ def get_entities(self):
         self._ensure_loaded()
         return list(self.entities.values())
 
-    def reload(self):
-        """Reload the ACL data from Cloud Storage.
+    @property
+    def reload_path(self):
+        """Compute the path for GET API requests for this ACL.
 
         This is a virtual method, expected to be implemented by subclasses.
 
-        :raises: :class:`NotImplementedError`
+        :raises: :class:`NotImplementedError` if ``_reload_path`` attribute
+                 is not set on the instance.
         """
+        # Allow override for testing
+        path = getattr(self, '_reload_path', None)
```

Comment:
The hoisted `reload` implementation in the base ACL class relies on having a `reload_path` property, which is implemented via a "pure virtual" method in the base class (raising `NotImplementedError`).  We cannot write a test for the `reload` implementation in that class (it *used* to be pure virtual too, but now uses the property).  We would either need to subclass to test it, or else allow tests to inject a value which overrides the pure virtualness.